docs/api/migration-guides/v2-primitives.mdx

```diff
@@ -0,0 +1,1202 @@
+
+## Overview
+
+Version 2 of the primitives is introduced with a new base class for both Sampler and Estimator ([BaseSamplerV2](/api/qiskit/qiskit.primitives.BaseSamplerV2) and [BaseEstimatorV2](/api/qiskit/qiskit.primitives.BaseEstimatorV2)), along with new types for their inputs and outputs. The emphasis of this new version is on performing vectorized calls to the `primitive run()` methods so that you can efficiently specify sweeps over parameter value sets and observables. See [StatevectorSampler](/api/qiskit/qiskit.primitives.StatevectorSampler) and [StatevectorEstimator](/api/qiskit/qiskit.primitives.StatevectorEstimator) for V2 primitive reference implementations.
+
+Based on user feedback, this version introduces the following major new functions:
+
+* A new interface that lets you specify a single circuit and multiple observables (if using Estimator) and parameter value sets for that circuit. Previously, you had to specify the  same circuit multiple times to match the size of the data to be combined.
+* SamplerV2 
+  - Returns the outputs (bitstrings) from every shot, rather than providing a counts-like return (quasi-probabilities). The result class, however, has methods to return weighted samples, such as counts.
+  - Returns data from separate `ClassicalRegisters`. This enables derived classes to implement Sampler support for circuits with classical control flows.
+  - No longer supports resilience levels (error mitigation).  
+* EstimatorV2 
+  - Has a new `precision` argument in the `run()` method that specifies the targeted precision of the expectation value estimates.
+  - Does not support resilience level 3.  This is because resilience level 3 in V1 Estimator uses Probabilistic Error Cancellation (PEC), which is proven to give unbiased results at the cost of exponential processing time.  You can, however, still use PEC as the error mitigation method by using the `pec_mitigation` option.
+* Both V2 primitives let you turn on or off individual error mitigation / suppression methods.
+
+<Admonition type="caution" title="Important">
+To ensure faster and more efficient results, as of 1 March 2024, circuits and observables need to be transformed to only use instructions supported by the system (referred to as *instruction set architecture (ISA)* circuits and observables) before being submitted to the Qiskit Runtime primitives.  See the [transpilation documentation](../../transpile) for instructions to transform circuits.  Due to this change, the primitives will no longer perform layout or routing operations.  Consequently, transpilation options referring to those tasks will no longer have any effect. Users can still request that the primitives do no optimization of input circuits by specifying `optimization_level=0`.
```

Comment:
I would add the term "transpilation" as part of the first sentence. I understand that we want to introduce the term ISA circuits but it is not widespread yet, and transpilation might hit the spot faster.

Reply:
I'll just remove it from here, since, as you point out, it's not a change.